lib/resource-providers/rds.ts

```diff
@@ -0,0 +1,55 @@
+import { CfnOutput } from "aws-cdk-lib";
+import * as rds from "aws-cdk-lib/aws-rds";
+import { GlobalResources, ResourceContext, ResourceProvider } from "../spi";
+import {IClusterEngine, IDatabaseCluster, InstanceProps} from "aws-cdk-lib/aws-rds";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import {IVpc} from "aws-cdk-lib/aws-ec2";
+
+
+export interface RdsInstanceProps {
```

Comment:
What is this interface for? is it WIP for something else?

Reply:
An RDS Provider should ideally also support RDS MySQL/RDS Postgres.
And specific database instances rather than just aurora clusters